geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/ExportStackTraceCommandDUnitTest.java

```diff
@@ -97,4 +104,40 @@ public void exportStackTrace_with_file() {
     // delete this file afterwards so that we won't pollute the other tests in this class
     files[0].delete();
   }
+
+  @Test
+  public void exportStackTrace_check_file_content() throws IOException {
```

Comment:
Can you change the method name to reflect what the test is doing and try to use _camelCase_ for the name instead of underscores?.
**_PD:_** I realise that other methods within the class use underscore, but I'd prefer us to use accepted formats when adding new code.